src/job.rs

```diff
@@ -1399,22 +1416,22 @@ mod tests {
 
     use crate::test_utils::TestContext;
 
-    async fn insert_job(context: &Context, foreign_id: i64) {
+    async fn insert_job(context: &Context, foreign_id: i64, valid: bool) {
```

Comment:
Why was `valid` parameter added? Why was it needed to change these tests? Previously they tested invalid `foreign_id`, now they test invalid `action`, why?